pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -79,29 +79,27 @@ public static String upper(String input) {
   /**
    * @see String#substring(int)
    * @param input Parent string
-   * @param beginIndex index from which substring should be created
+   * @param beginIndex 1 based index from which substring should be created
    * @return substring from beginIndex to end of the parent string
    */
   @ScalarFunction(names = {"substr", "substring"})
```

Comment:
Shall we consider keep `substr` as current behavior and use `substring` for postgresql behavior?